pywinauto/controls/HwndWrapper.py

```diff
@@ -163,6 +163,10 @@ class HwndWrapper(object):
     can_be_label = False
     has_title = True
 
+    # specify whether we need to grab an image of ourselves
+    # when asked for properties
+    _NeedsImageProp = False
```

Comment:
@moden-py, I'm not sure that I understand it. Why do you declare *_NeedsImageProp* as a class attribute, but later it's used as a property of an instance of the class ? You could decorate it as described here: https://www.quora.com/How-do-I-create-a-class-property-in-Python:
```python
@property
def NeedsImageProp():
    return ButtonWrapper._NeedsImageProp
```
However, it seems as not your case. You calculate the value according to instance's *style* variable. Isn't changing the attribute value affects all instances ?